app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -164,19 +149,48 @@ Set up a Teaching Vacancies job alert to be updated about opportunities througho
+
+## Plan your move to the UK
+
+The following resources may be useful as you plan your move:
+
+* [open a digital bank account](https://www.moneysupermarket.com/current-accounts/digital-banking/) (you can do this before you arrive in the UK, and you will not need a UK address to do so)
+* [find somewhere to live](https://www.gov.uk/government/publications/how-to-rent)
+* [get a national insurance (NI) number](https://www.gov.uk/national-insurance/your-national-insurance-number)
+* [register with a doctor](https://www.nhs.uk/nhs-services/gps/how-to-register-with-a-gp-surgery/)
+* [find childcare](https://www.gov.uk/get-childcare)
+* [learn about tax](https://www.gov.uk/income-tax)
+* learn about [indefinite leave to remain](https://www.gov.uk/indefinite-leave-to-remain)
 
-Schools have some flexibility on [teachers' salaries](/salaries-and-benefits). It varies by region and your level of skill, experience and qualifications.
 
-There are [specific salary requirements for a skilled worker visa](https://www.gov.uk/government/publications/teach-in-england-if-you-qualified-outside-the-uk/teach-in-england-if-you-qualified-outside-the-uk#visas-and-immigration).
 
 ## Learn about life as a teacher in England
 
+If you’ve successfully used the [Apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) service, and have less than 2 years’ teaching experience, you can get valuable support and mentoring through the [early career framework](https://www.gov.uk/government/publications/early-career-framework). 
+
 [Read other teachers’ stories](/blog) and research the [English curriculum](https://www.gov.uk/national-curriculum) and [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards).
 
+### Meet non-UK citizens who have made the move to teach in England 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/owLXaeerFy8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
 ## Contact
 
-The Department for Education does not administer teacher training courses or appoint teachers, so we cannot help with individual applications to training providers or schools.
+The Department for Education (DfE) does not employ teachers directly. Do not send us your CV, as we will not be able to help you find a teaching job. 
+
+Please do not email to ask when the country where you qualified as a teacher or your subject specialism will be added to the [Apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) service. We’ll publish that information on [A fairer approach to awarding QTS to overseas teachers](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers/a-fairer-approach-to-awarding-qts-to-overseas-teachers--2#) as new countries and subjects are added.
```

Comment:
Suggestion:

Please do not email to ask when a country or subject will be added to the list for QTS in England. We’ll [publish updates as new countries and subjects are added.](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers/a-fairer-approach-to-awarding-qts-to-overseas-teachers--2#)

Reply:
yes that's better